internal/pkg/cli/job_init.go

```diff
@@ -158,10 +160,23 @@ func (o *initJobOpts) Validate() error {
 
 // Ask prompts for fields that are required but not passed in.
 func (o *initJobOpts) Ask() error {
-	if err := o.askJobType(); err != nil {
+	if err := o.askJobName(); err != nil {
 		return err
 	}
-	if err := o.askJobName(); err != nil {
+	localMft, err := o.mftReader.ReadWorkloadManifest(o.name)
+	if err == nil {
+		jobType, err := localMft.WorkloadType()
+		if err != nil {
+			return fmt.Errorf("read type for service %s from local manifest: %w", o.name, err)
+		}
+		o.wkldType = jobType
+		return nil
+	}
+	_, ok := err.(*workspace.ErrFileNotExists)
+	if !ok {
```

Comment:
in order to work with wrapped errors:
```suggestion
	var errNotFound *ErrFileNotExists
	if !errors.As(err, &errNotFound) {
```